app/support/products/scheduling_support.rb

```diff
@@ -94,12 +94,16 @@ def available_schedule_rules(user)
     end
   end
 
-  def online!
-    offline_reservations.current.update_all(reserve_end_at: Time.current)
+  def offline?
+    offline_reservations.current.any?
+  end
+
+  def offline_via_schedule_share?
```

Comment:
I wanted a distinction between being offline and being offline by proxy, particularly on the admin side.

I agree the behavior is awkward. If we need to back out and rethink, the commits are isolated.